static/src/javascripts-legacy/projects/commercial/modules/dfp/on-slot-render.js

```diff
@@ -34,6 +34,7 @@ define([
             reportEmptyResponse(advert.id, event);
             emitRenderEvents(false);
         } else {
+            advert.size = event.size;
```

Comment:
Yes, it is documented here: https://developers.google.com/doubleclick-gpt/reference#googletageventsslotrenderendedevent